hack/prefetch-images.sh

```diff
@@ -0,0 +1,61 @@
+set -ex pipefail
+
+source hack/common.sh
+source cluster-up/cluster/$KUBEVIRT_PROVIDER/provider.sh
+source hack/config.sh
+
+function prefetch-images::find_node_names() {
+    if [[ $KUBEVIRT_PROVIDER == "external" ]] || [[ $KUBEVIRT_PROVIDER =~ kind.* ]] || [[ $KUBEVIRT_PROVIDER == "local" ]]; then
```

Comment:
This condition is duplicated, can you wrap it in a function? (if you really need the other one)

Reply:
actually the other one not needed, you are right